lib/checkother.cpp

```diff
@@ -3114,6 +3114,8 @@ void CheckOther::checkNegativeBitwiseShift()
     logChecker("CheckOther::checkNegativeBitwiseShift");
 
     for (const Token* tok = mTokenizer->tokens(); tok; tok = tok->next()) {
+        skipUnreachableBranch(tok);
```

Comment:
I think `tok = skipUnreachableBranch(tok);` would be more readable.

Reply:
Yep, on it